gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -4,6 +4,7 @@
 {% import "%namespace/%name_%version/%sub/services/%service/_client_macros.j2" as macros %}
 {% import "%namespace/%name_%version/%sub/services/%service/_shared_macros.j2" as shared_macros %}
 
+import logging as std_logging
```

Comment:
I'm curious: why are you renaming the standard import? We could simply use non-conflicting names for our own constructs/modules.

Reply:
There was a naming conflict in the logging goldens:

```
import logging
from google.cloud.logging_v2.types import logging
```

Reply:
I files https://github.com/googleapis/gapic-generator-python/issues/2269 for the long term fix which is to avoid generating modules, [like this](https://github.com/googleapis/gapic-generator-python/blob/main/tests/integration/goldens/logging/google/cloud/logging_v2/types/logging.py), which conflict with [Python3 built-in modules](https://docs.python.org/3/py-modindex.html)

Reply:
One alternative would be to add `logging` to the reserved names list [here](https://github.com/googleapis/gapic-generator-python/blob/d87480974bf230647e77343f30b1c82a08773a03/gapic/utils/reserved_names.py#L32).

However doing so would result in a breaking change in other libraries such as `google-cloud-build` where field with the name `logging` exists.
https://github.com/googleapis/google-cloud-python/blob/adea041c1d43b94d11ff9d4d5ba13c0d7898cac2/packages/google-cloud-build/google/cloud/devtools/cloudbuild_v1/types/cloudbuild.py#L3368

The field `logging` would be renamed to `logging_`, similar to how `license` was renamed to `license_`
https://github.com/googleapis/gapic-generator-python/issues/2015

The reason is that the reserved terms list is tightly coupled with the logic to alias import statements, and avoid collisions in field names. IOW, we don't have separate lists for just adding an alias to import statements.

https://github.com/googleapis/gapic-generator-python/blob/d87480974bf230647e77343f30b1c82a08773a03/gapic/schema/api.py#L175-L181

https://github.com/googleapis/gapic-generator-python/blob/d87480974bf230647e77343f30b1c82a08773a03/gapic/schema/wrappers.py#L78-L82